### PR TITLE
[SCB-595]Compatible with old SC

### PR DESCRIPTION
**Pull request by laijianbin**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by WillemJiang** [comment]
There are too many commits, please squash them to a single one.

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java [hunk 1]
```diff
@@ -184,6 +184,16 @@ private void loadFrameworkVersions() {
...
     microservice.setRegisterBy(CONFIG_DEFAULT_REGISTER_BY);
+    ServiceCenterInfo info = srClient.getServiceCenterInfo();
+    while (info == null) {
+      return;
+    }
+    String serviceCenterVersion = info.getVersion();
+    String validateSuccessVersion = "1.0.0";
+    int compareResult = serviceCenterVersion.compareTo(validateSuccessVersion);// 如果大于1.0.0，不需要兼容
+    if (compareResult < 1) {
+    microservice.getFramework().setVersion(null);
```
[the remark this answers is not in the document]
Comment (liubao68) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java]: In last PR, we have removed this logic, https://github.com/apache/incubator-servicecomb-java-chassis/pull/717/files. What's the difference between this modification? I think they got almost the same issue.

[the remark this answers is not in the document]
Comment (laijianbin) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java]: @WillemJiang In the beginning，ServiceComb registers microservice  version like this "version":"null", but after this PR(https://github.com/apache/incubator-servicecomb-java-chassis/pull/588/files) ServiceComb registers microservice  version like this"version":"ServiceComb:null",this change cause a problem (
[2018-06-01 16:17:03,887/GMT+08:00][registry-vert.x-eventloop-thread-0][WARN]get response for org.apache.servicecomb.serviceregistry.api.response.CreateServiceResponse failed, 400:Bad Request, {"errorCode":"400001","errorMessage":"Invalid parameter(s)","detail":"Version validate failed, rule: {Length: 64,Length: ^[a-zA-Z0-9_\\-.]*$}"}
)when use old SC (2.2.49)

**Comment by wujimin** [comment]
1.change last commit, you can use "amend last commit"
2.change not last commit, you can use "rebase"

it's better to make every commit meaningful

**Comment by laijianbin** [comment]
As there are too many PR,I close this PR and commit another PR

**Pull request closed without merging**